### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Manankin**
[DEMO LINK](https://manankin.github.io/js_task_generate_table_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -355,6 +355,33 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people); // you can remove it
```
Comment (DanilWeda) [inline src/scripts/main.js:358]: Think, you can remove it, also as eslint comment

**Review by DanilWeda — CHANGES_REQUESTED** [review]
Good job. 
Check some small comments. 
Thanks)

File: src/scripts/main.js [hunk 2]
```diff
@@ -355,6 +355,33 @@ const people = [
...
 
 // write your code here
+const table = document.querySelector('.dashboard');
+// console.log(table);
+
+people.forEach((person) => {
+  // eslint-disable-next-line no-shadow
+  const { name, sex, born, died } = person;
```
[the remark this answers is not in the document]
Comment (Manankin) [reply src/scripts/main.js:366]: > Also, you can make destructuring on forEach callback ( not critical )

Yes, I know it. But I prefer to make destructing outside of argument, it's more readable for me.

**Review by DanilWeda — APPROVED** [review]
Great. 
Approved. 
Thanks)

**Pull request closed without merging**